routes/financial/financial.spec.js

```diff
@@ -29,4 +29,12 @@ describe('Test /financial responses', () => {
     expect(response.statusCode).toBe(302)
     expect(response.headers.location).toEqual('/deductions/rrsp')
   })
+
+  test('it redirects to the checkAnswers when posting Yes and having come from the checkAnswers page', async () => {
+    const response = await request(app)
+      .post('/financial/income')
+      .send({ confirmIncome: 'Yes', redirect: '/deductions/rrsp' }).query({ref: 'checkAnswers'})
```

Comment:
Interesting, so I guess `.query` doesn't have to come directly after `.post`?

Reply:
oops, good catch!